code/obj/item/sec_tape.dm

```diff
@@ -0,0 +1,162 @@
+#define MAXTAPE 40
+#define STARTAMOUNT 1
+/// The security tape roll object, used to create security tapes.
+/obj/item/sec_tape
+	name = "security tape roll"
+	var/base_name = "security tape roll"
+	desc = "A roll of security tape."
+	amount = STARTAMOUNT
+	max_stack = MAXTAPE
+	stack_type = /obj/item/sec_tape
+	icon = 'icons/obj/sec_tape.dmi'
+	icon_state = "sec_tape_roll"
+	throwforce = 2
```

Comment:
Why set all these combat values for an item you are not meant to hit people with?

Reply:
You're right, removed them